tests/summarizer/test_image_description_retreiver.py

```diff
@@ -116,3 +116,64 @@ def test_bad_request(mocked_post):
         image_describer.get_description_for_images(["bad_url"])
         assert error.message \
             == "The API call was unsuccessful with status code: 400"
+
+
```

Comment:
Should also check if request_number is returned properly by make_post_request.

Reply:
Updated accordingly.